client/src/pages/Apply/Apply.js

```diff
@@ -95,13 +96,16 @@ const Apply = ({ navigation }) => {
     if (acceptTerms === false) {
       validateErrors.acceptTerms = 'Terms are required'
     }
```

Comment:
Will need to remove this since it doesn't matter if the followUp is checked or now. I would also suggest updating the conditional statements above so they don't check `someExpression === false`. `!someExpression` is better syntax.

Reply:
Actually, ignore this comment - see the comment below.